ansible/group_vars/all.yml

```diff
@@ -96,3 +96,21 @@ dns_replicas: 1
 
 # See kube documentation for apiserver runtime config options.  Example below enables HPA, deployments features.
 # apiserver_extra_args: "--runtime-config=extensions/v1beta1/deployments=true"
+
```

Comment:
Weird, take them out then. We need to make sure this PR does not change these params.